src/components/pages/Admin/components/RegistrationRequests/registrationTableColumns.tsx

```diff
@@ -18,62 +18,181 @@
 
 // Columns definitions of Registration Request page Data Grid
 export const RegistrationRequestsTableColumns = (
   t: typeof i18next.t,
-  handleDownloadDocument: (
-    appId: string,
-    documentId: string,
-    documentType: string
-  ) => void,
   showConfirmOverlay?: (applicationId: string) => void,
-  onConfirmationCancel?: (applicationId: string, name: string) => void,
-  onChipButtonSelect?: (button: ProgressButtonsType, id: string) => void
+  onConfirmationCancel?: (applicationId: string, name: string) => void
 ): Array<GridColDef> => {
-  return [
-    {
-      field: 'dateCreated',
-      headerName: t('content.admin.registration-requests.columns.date'),
-      flex: 1.5,
-      disableColumnMenu: true,
-      valueGetter: ({ row }: { row: ApplicationRequest }) =>
-        dayjs(row.dateCreated).format('YYYY-MM-DD'),
-    },
+  const getStatusProgress = (row: ApplicationRequest) => {
+    const todoItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.TO_DO
+    ).length
+    const inprogressItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.IN_PROGRESS
+    ).length
+    const doneItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.DONE
+    ).length
+    const failedItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.FAILED
+    ).length
+
+    let items = {
+      TO_DO: todoItems,
+      IN_PROGRESS: inprogressItems,
+      DONE: doneItems,
+      FAILED: failedItems,
+    }
+
+    const getProgressStatus = (
+      style: React.CSSProperties,
+      row: ApplicationRequest,
+      statusText: string
+    ) => {
+      return (
+        <div
+          className="progressMain"
+          style={{
+            border: `2px solid ${style.border}`,
+            background: style.background,
+          }}
+        >
+          <Progress
+            applicationStatus={row.applicationStatus}
+            items={items}
+            totalItems={row.applicationChecklist.length}
+          />
+          <Typography
+            variant="body2"
+            className="statusText"
+            sx={{ color: style.color }}
+          >
+            {statusText}
+          </Typography>
+        </div>
+      )
+    }
 
+    if (row.applicationStatus === ApplicationRequestStatus.SUBMITTED) {
+      const style = {
+        border: items.FAILED ? '#d91e18' : '#EAF1FE',
+        color: items.FAILED ? '#d91e18' : '#000',
+      }
+      return (
+        <div className="statusMain">
+          {getProgressStatus(
+            style,
+            row,
+            items.FAILED
+              ? t('content.admin.registration-requests.buttonerror')
+              : t('content.admin.registration-requests.buttonprogress')
```

Comment:
put ternary into the string
`t('path.to.whatever.button${items.FAILED ? 'error' : 'progress'}')`